env.sh

```diff
@@ -2,12 +2,12 @@
 
 # TODO: Specify these variables according to your system configuration!
 # the user running the application
-export DDUSER=
+export DDUSER=${DD_APPUSER-${USER}}
```

Comment:
I'd prefer
<pre>
`whoami`
</pre>
to 
<pre> ${USER}$</pre>
if possible...
https://groups.google.com/forum/#!topic/comp.unix.shell/PLZPzNx3F_4